src/hats/catalog/dataset/dataset.py

```diff
@@ -66,6 +58,5 @@ def _read_kwargs(cls, catalog_base_dir: str | Path | UPath) -> dict:
     def _check_files_exist(cls, catalog_base_dir: str | Path | UPath):
         if not file_io.does_file_or_directory_exist(catalog_base_dir):
             raise FileNotFoundError(f"No directory exists at {str(catalog_base_dir)}")
-        catalog_info_file = paths.get_catalog_info_pointer(catalog_base_dir)
-        if not file_io.does_file_or_directory_exist(catalog_info_file):
-            raise FileNotFoundError(f"No catalog info found where expected: {str(catalog_info_file)}")
+        if not file_io.does_file_or_directory_exist(catalog_base_dir / "properties"):
```

Comment:
So the `properties` file doesn't have an extension?  Or does `file_io` imply one?

Reply:
Answered my own question later in the review: it doesn't! 👍